assets/js/validation.js

```diff
@@ -16,6 +16,30 @@ const validateName = function () {
             validationResult.innerText = 'Error: ' + error;
         });
 };
-
 name.onkeyup = validateName;
 name.onchange = validateName;
+
+document.getElementById('komanda').addEventListener("click", komanda(), true);
```

Comment:
Lankytojai komandos laukelį gali pasiekti gali pasiekti ne tik su pele, bet ir naudojant `tab` klavišą.
Todėl tavo `addEventListener("click"` ne visada (kada reikia) būtų iškviečiamas.

Mano pavyzdyje laukeliai turi tik po vieną funkciją (angl. _listener_). todėl ar rašiau tiesiogiai
```javascript
name.onkeyup = validateName;
name.onchange = validateName;
```
Kas atitiktų
```javascript
name.addEventListener("keyup", validateName);
name.addEventListener("change, validateName);
```

`validation.js` failas yra [įdedamas puslapio apačioje](https://github.com/Romienas/kickstart/blob/351c090c55fc3ea5194d45d1694976086eca58d4/templates/people/index.html.twig#L25)
```twig
 {{ encore_entry_script_tags('validation') }}
```
todėl visi elementai jau yra sukurti ir galima jiems tiesiogiai uždėti `key`, `change` ar `click` event listener'ius.
> Analogija JQuery `$(document).ready(`

Plačiau:
 * https://www.w3schools.com/jsref/met_element_addeventlistener.asp|
 * https://www.w3schools.com/jsref/event_onclick.asp

Reply:
Nors daug kur pavyzdžiuose naudojama `komanda()` (su skliaustais). Verta pradėti skirti funkcijos įvykdymą ir nuorodą į ją.
```suggestion
document.getElementById('komanda').addEventListener("click", komanda, true);
```

Nes kai pradėsi dirbti su [Promise](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Promise), tada bus daug painiavos.

Pavyzdžiui:
```javascript
function daugyba(a, b) {
    console.log("Dauginu", a, b);
    return a * b;
}
function suma(a, b) {
    console.log("Sudedu", a, b);
    return a + b;
}

function ivykdyk(veiksmas, a, b) {
    console.log("Vykdau veiksmą", veiksmas, "skaičiamas", a, b);
    return veiksmas(a, b);
}

let pirmas = ivykdyk(daugyba, 2, 3); // <-- Be skliaustų
console.info("Pirmas: ", pirmas);
let antras = ivykdyk(suma, 2, 3);  // <-- Be skliaustų, galima paduoti kitą funkciją
console.info("Antras: ", antras);
let trecias = ivykdyk(daugyba(), 2, 3);  // <-- Klaida, kai duoda reikšmę, bet tikimasi funkcijos
console.info("Trečias ", trecias);
```
![image](https://user-images.githubusercontent.com/1453957/57179416-7343b180-6e86-11e9-847d-aacbd22f5257.png)

Plačiau:
 * https://developer.mozilla.org/en-US/docs/Glossary/Callback_function
 * https://medium.freecodecamp.org/a-quick-intro-to-higher-order-functions-in-javascript-1a014f89c6b